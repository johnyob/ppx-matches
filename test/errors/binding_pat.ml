(* Error: patterns that bind variables are not allowed in inline form without when clause *)
let value = Some 42
let () = assert ([%matches? Some x] value)
