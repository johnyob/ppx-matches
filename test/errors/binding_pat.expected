File "binding_pat.ml", line 3, characters 16-42:
Error: ppx_matches: invalid [%matches? ...] payload. Pattern `Some
       x` cannot bind any variables.
